game/src/main/scala/io/aigar/game/EntityContainer.scala

```diff
@@ -19,19 +23,24 @@ trait EntityContainer {
     None
   }
 
-  def handleCollision(entities: List[Entity], players: List[Player]): List[Entity] ={
+  def handleCollision(entities: List[Entity],
+                      players: List[Player],
+                      scoreModifications: Option[MutableList[ScoreModification]]): List[Entity] ={
```

Comment:
Rather than `MutableList`s, could we go with [multiple return values via tuples](http://alvinalexander.com/scala/how-to-define-methods-return-multiple-items-tuples-scala)?

Another solution would be to go back to a `List[ScoreModification]` return type and move the management of the `entities` list in `EntityContainer` (*it* would take care of removing it from the list, for example).